op-proposer/proposer/service.go

```diff
@@ -139,6 +141,12 @@ func (ps *ProposerService) initRPCClients(ctx context.Context, cfg *CLIConfig) e
 		return fmt.Errorf("failed to build L2 endpoint provider: %w", err)
 	}
 	ps.RollupProvider = rollupProvider
+
+	eotsAggClient, err := eotsclient.NewEotsAggregatorGRpcClient(cfg.AggregatorRpc)
+	if err != nil {
+		return fmt.Errorf("failed to create EOTS aggregator client : %w", err)
+	}
```

Comment:
It seems that the `eotsAggClient` does not share the context `ctx` with other clients. This means if we kill the program via `Kill()` the EOTS client will still be alive. Will this be a problem?

Reply:
Yes, updated the gRPC client to accept a `Context` parameter.